tools/transformer/src/engine/readers.js

```diff
@@ -118,6 +118,8 @@ function getNameFromYamlPath(path) {
 }
 
 function getObjectFromYamlPath(path, obj) {
+  errMessage = `Your schema contain inline openapi objects. 
+  This usually happens when schema contains oneOf or anyOf statements directly in the response or request body.`;
```

Comment:
Nit; Worth including a suggestion to mitigate against this specific issue?

Reply:
I think I have provided an generic open API mitigation. Not sure if that would not confuse backend engineers? looking for feedback. Would you understand what you need to do?